cmake/SSGCommon.cmake

```diff
@@ -695,7 +692,7 @@ macro(ssg_build_product PRODUCT)
         generate-ssg-${PRODUCT}-xccdf-1.2.xml
         generate-ssg-${PRODUCT}-oval.xml
         generate-ssg-${PRODUCT}-ocil.xml
-        generate-ssg-${PRODUCT}-cpe-dictionary.xml
+        generate-ssg-cpe-dictionary.xml
```

Comment:
Should I remove the dependency on cpe dictionary for product builds? I renamed the target as we don't build it per platform anymore